packages/list/ListItem.tsx

```diff
@@ -90,21 +165,51 @@ export default class ListItem<T extends HTMLElement = HTMLElement> extends React
       checkboxList,
       radioList,
       onDestroy,
+      onClick,
```

Comment:
I'm confused, what is the difference between onClick and handleClick ?

Reply:
see https://github.com/material-components/material-components-web-react/pull/786/files/f93bcdf660f50ff5a4ce61e65287b25520df2d1c#r279107245

Reply:
:+1: